WORKSPACE

```diff
@@ -63,6 +63,7 @@ load(
     )
     for arch in ARCHITECTURES
     for (name, distro) in VERSIONS
+    if "security" in SHA256s[arch][name]
```

Comment:
There's no security mirror...?

Reply:
security.debian.org does not have s390x for debian stretch:
http://security.debian.org/debian-security/dists/stretch/updates/main/
and there are no any other mirrors that include s390x.